Tiltfile

```diff
@@ -28,6 +28,8 @@ manager_yaml = decode_yaml_stream(kustomize("config/default"))
 for resource in manager_yaml:
     if resource["metadata"]["name"] == "capl-manager-credentials":
         resource["stringData"]["apiToken"] = os.getenv('LINODE_TOKEN')
+    if resource["kind"] == "CustomResourceDefinition":    
+        resource["metadata"]["labels"]["clusterctl.cluster.x-k8s.io"] = ""
```

Comment:
This could be overly broad, could we instead limit this to `spec.group=infrastructure.cluster.x-k8s.io` perhaps?

Reply:
Switched it :)